calculator4.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
+    return x + y
+
+def subtract(x, y):
```

Comment:
Consider using docstrings to provide a brief explanation of what each function does. This will make the code easier to understand for other developers. [medium]